app/service/group_service.rb

```diff
@@ -14,26 +14,43 @@ def upgrade_group
 
   def request_upgrade
     @group.upgrade_requested!
+    send_group_upgrade_requested_emails
   end
 
 private
 
   def send_group_upgraded_emails
     @group.memberships.each do |membership|
-      send_group_updated_email(membership.user) if notify_member?(membership)
+      send_group_updated_email(membership.user.email) if notify_member?(membership)
```

Comment:
Should this change have been in the previous commit?

Reply:
It could be, but not sure its worth the effort moving it. It's related to the changes here enough and is isolated so I don't think it's too difficult to follow?